tests/infra/network.py

```diff
@@ -1178,31 +1201,29 @@ def wait_for_snapshot_committed_for(self, seqno, timeout=3):
         return self.wait_for_commit_proof(primary, snapshot_evidence_seqno, timeout)
 
     def get_committed_snapshots(self, node):
-        # Wait for all available snapshot files to be committed before
-        # copying snapshot directory, so that we always use the latest snapshot
+        # Wait for the snapshot including target_seqno to be committed before
```

Comment:
This logic assumed that no other transaction were running concurrently. Obviously, this isn't true anymore.